plugin/src/sbt-test/sbt-mu-srcgen/basic/build.sbt

```diff
@@ -5,3 +5,5 @@ enablePlugins(SrcGenPlugin)
 libraryDependencies ++= Seq(
   "io.higherkindness" %% "mu-rpc-server" % sys.props("mu")
 )
+
+addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.patch)
```

Comment:
I added this plugin because the `sbt-test`s were failing on with an error about not being able to evaluate the `@service` macro that we're using in our source generation.  It seemed to work for the `basic` tests, but for some reason the `srcGenFromDirs` and `srcGenFromJars` are still failing on the `sbt-test` component of the build.  I'll look into it more on Monday but if anyone knows what could be causing these errors, that would be awesome.

Reply:
Ooooh, I think I figured it out;  I dropped a conditional on 155 of `AvroSrcGenerator.scala` that was critical to not writing useless template code when an Avro schema doesn't have any methods in it.  Tests passed locally; let's see if they work in CI...

Reply:
So this line can be removed?

Reply:
Ah, yes!  Missed this last one.